packages/website-frontend/test/unit/pages/home.page.spec.ts

```diff
@@ -45,6 +45,6 @@ describe(HomePage.name, () => {
     const link = sut.element.querySelector('sme-hero')?.shadowRoot?.querySelector('sme-link');
 
     // Assert
-    expect(link?.href).to.eq('#getting-started')
+    expect(link?.href).to.eq('#getting-started');
```

Comment:
It seems like ESLint does not know about the custom elements.

We might need to add `as CustomElement` to this code, not nice though.

Reply:
I managed to fix it. The issue was a tsconfig setup where it tried to compile the frontend before building elements